docs/en/index.md

```diff
@@ -0,0 +1,341 @@
+---
+title: Silverstripe elemental
+---
+
+# Silverstripe elemental
+
+## Introduction
+
+This module extends a page type to swap the content area for a list of manageable elements to compose a page out
+of rather than a single text field. Features supported:
+
+- Versioning of elements
+- Ability to add, remove supported elements per *elemental area*
+
+The module provides basic markup for each of the elements but you will likely need to provide your own styles. Replace
+the `$Content` variable with `$ElementalArea` in your page templates, and rely on the markup of the individual elements.
+
+For a more detailed overview of using this module, please see [the User help guides](docs/en/userguide/index.md).
+
+## Requirements
+
+- Silverstripe CMS ^5.0
+- Versioned Admin ^2.0
+- GridFieldExtensions ^4.0
+
+For a CMS 4.3+ compatible version of this module, please see the [4.x release line](https://github.com/silverstripe/silverstripe-elemental/tree/4#readme).
+
+## Installation
+
+```bash
+composer require dnadesign/silverstripe-elemental
+```
+
+The following YAML config will enable elements on every `Page` object,
+replacing the standard `Content` rich text field.
+
+```yml
+# app/_config/elements.yml
+Page:
+  extensions:
+    - DNADesign\Elemental\Extensions\ElementalPageExtension
+```
+
+In your page type layout template use `$ElementalArea` to render the elements to the page (in place of `$Content`).
+
+## Getting more elements
+
+Note that this module comes by default with the base element and a "Content" element. If you need more, take
+a look at some other modules:
+
+## Silverstripe CMS supported content block modules
+
+To learn more about [Silverstripe CMS supported](https://www.silverstripe.org/software/addons/supported-modules-definition/) content block types see, [Creating new blocks](/docs/en/userguide/edit_content.md).
```

Comment:
That definition is way outdated. Just link to https://docs.silverstripe.org/en/project_governance/supported_modules/ instead.

Reply:
Also the userdoc link will need to link to the userdocs site explicitly - dev docs won't know how to resolve that path.

Reply:
Updated

Reply:
Updatedu